app/controllers/hiring_staff/base_controller.rb

```diff
@@ -1,32 +1,43 @@
 class HiringStaff::BaseController < ApplicationController
   before_action :authenticate
+  DEFAULT_SCHOOL_URN = '140181'.freeze
 
   def authenticate
     return unless authenticate_hiring_staff?
     authenticate_or_request_with_http_basic('Hiring Staff') do |name, password|
-      name == http_user && password == http_pass
+      if name == benwick_http_user && password == benwick_http_pass
+        session[:urn] = '110627'
+        true
+      elsif name == http_user && password == http_pass
+        session[:urn] = DEFAULT_SCHOOL_URN # Enables the product team to access publishing and the test suite a default
+        true
+      end
     end
   end
 
+  def current_school
+    @current_school ||= School.find_by! urn: session[:urn]
```

Comment:
I have been using `id`, which is our identifier, and not `urn`. This will still work for now but we should decide what to use as our identifying key for the admin facing and the user facing controllers

Reply:
I can switch to our school ID that's a better idea.

Reply:
I've remembered why I didn't use the school ID: they are always going to be different in each environment so they aren't something we can pin down reliably tie to a log in. The only identifying data that can works must come from Get Information About Schools.